mods/cnc/rules/misc.yaml

```
@@ -7,31 +7,39 @@ CRATE:
 	Crate:
 		Duration: 6000
 	GiveCashCrateAction:
+		TextNotification: Crate collected.
 		Amount: 1000
 		SelectionShares: 20
 		UseCashTick: true
 	RevealMapCrateAction:
+		TextNotification: Crate collected.
```

Being more specific here

```suggestion
		TextNotification: Map reveal crate collected.
```

might help with understanding the wonky crate feature better (and would be a reason to stay I think)

---

I don't have a strong opinion on this but I feel that it better be left to a follow up as this PR is already quite big.

---

This is in the testcase commit. I agree that it is better to go over all the texts in a follow-up PR.